index.html

```diff
@@ -302,17 +301,24 @@ <h1 class="contat">Contact me</h1>
                             a project that needs coding. I’d love to help with it </p>
                     </div>
 
-                    <div class="form">
-                        <input type="text" id="fname" class="form1" name="firstname" placeholder="Yeremias">
-                        <input type="text" id="lname" name="lastname" placeholder="natajayanj14@gmail.com">
-                        <textarea id="subject" name="subject" class="form3" placeholder="Write your message here"
-                            style="height:90px"></textarea>
-
-                    </div>
-
-                    <div class="form-button">
-                        <button class="see">Get In Touch</button>
-                    </div>
+                    <footer>
+
+                        <form class="form1" action="https://formspree.io/f/xgeqypdd" method="post">
+                            <ul class="widgets">
+                                <li><input type="text" id="fname" class="form1" name="firstname" maxlength="30"
+                                        placeholder="Abdimulhin" required></li>
+                                <li><input type="text" id="email" name="email" placeholder="adamabdimu@gmail.com"
+                                        required></li>
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 It looks like you still haven't done what the previous code reviewer said, don't worry it's easy to do. Please change `type="text"` ❌ to `type="email"` ✔️(for line `310`).